### DISCUSSION TITLE
I cannot find DATASET tab in 7.0.3 release and the latest souce code

### DISCUSSION THREAD

**Discussion by yangxj1976** [TestOptimal/MBT#8] [Q&A] [answered]
Is this feature only provided in the ProMBT version?

**Comment by testoptimal-support** [comment, marked as answer]
The "DATASET" tab was removed from version 7.0.3 as part of the major overhaul effort to re-focus TestOptimal/MBT and ProMBT.  It's replaced by MScript plugin "$DATA" for simplicity.
